src/runtime/logging/ProtocolLogs.ts

```diff
@@ -0,0 +1,26 @@
+import { ConfigLogs } from "./types";
+import { LoggingConfig } from "./LoggingConfig";
+
+export class ProtocolLogs {
+  public get config(): ConfigLogs | undefined {
+    return this._config;
+  }
+
+  public get getConfig(): ConfigLogs {
+    if (!this._config) this._config = new ConfigLogs();
+    return this._config;
+  }
+
+  private _config?: ConfigLogs;
+
+  public arcError(error: Error) {
```

Comment:
Having "arc" here breaks the abstraction. Since this module is in the `runtime/` folder, it should have no reference to anything protocol specific (arc, daos, members, etc etc), just the abstractions the protocol uses (component, component list, protocol, etc)